presto-main/src/test/java/io/prestosql/cost/TestSemiJoinStatsCalculator.java

```diff
@@ -51,7 +51,7 @@
     private Symbol unknown = new Symbol("unknown");
     private Symbol fractionalNdv = new Symbol("fractionalNdv");
 
-    @BeforeMethod
+    @BeforeClass
```

Comment:
This is correct, but we could make these final and initialize them in the field declarations. I also noticed all the symbols in this class could be final.

Reply:
Let me consider this out of scope for this PR